packages/swarm-components/src/testUtils/screenRenderer.js

```diff
@@ -111,6 +111,10 @@ class screenRenderer {
 		this.log(`Testing: ${testUrl}`);
 		await page.goto(testUrl);
 
+		if (screenshotConfig.beforeScreenshot) {
```

Comment:
this looks unused now

Reply:
Yep, but can be used in the future as needed ;)

Reply:
hm, how can we document it so it's not a hidden feature? Not sure anyone would realize this is here.

Reply:
Added the docs